exercise1/exercise7/main.cpp

```diff
@@ -0,0 +1,26 @@
+/*Exercise 7 - Operator Precedence
+Write a program that will ask the user for three numbers (a,b,c) and output the following evaluations:
+a + 1 * b + 2 - c
+(a + 1) * (b + 2) - c
+*/
+#include <iostream>
+using namespace std;
+
+int main() 
+{
+	int a, b, c, x1, x2;
```

Comment:
Be mindful of your names. What are x1 and x2? Could they have more descriptive names?